app/controllers/user_filters_controller.rb

```diff
@@ -13,10 +13,12 @@ def create
     redirect_to root_url unless current_user.user_filters.count < 100 # Enforce upper limit per user
     active_filter_options = params.require(:activeFilterOptions).collect do |filter|
       {
-        filterOption: filter.require(:filterOption).permit(:name, :title, :description, :type, options: []),
+        filterOption: filter.require(:filterOption).permit(:name, :title, :description, :type, :hasTimestamp, :allowRange, options: []),
         value: filter.permit(:value)[:value] || filter.require(:value) || false,
+        numberOption: filter.permit(:numberOption)[:numberOption],
         dateOption: filter.permit(:dateOption)[:dateOption],
-        relativeOption: filter.permit(:relativeOption)[:relativeOption]
+        relativeOption: filter.permit(:relativeOption)[:relativeOption],
+        optionalOption: filter.permit(:optionalOption)[:optionalOption]
```

Comment:
Yeah I agree but couldn't come up with anything better and tried to stay with the naming scheme of the other options.  Any suggestions?  I broke this out so if we have any other filters that require an additional select, it would be extremely easy to add and makes the code cleaner.  I was going to use this for the "Contains" and "Exact Match" options for the Close Contact of Known Case filter - so there are other cases where this will be used

Reply:
That's good to know. Thoughts on `additionalFilterOption`? Still pretty vague but at least a little better.

Reply:
that's definitely better - I'll change it to that and if we ever think of something better we can change it again

Reply:
all set